cli_tools/gce_vm_image_import/importer/importer_test.go

```diff
@@ -247,56 +247,109 @@ func TestFlagsInvalidOS(t *testing.T) {
 	}
 }
 
-func TestBuildDaisyVarsFromDisk(t *testing.T) {
+func TestBuildDaisyVars(t *testing.T) {
 	resetArgs()
-	imageName = "image-a"
-	noGuestEnvironment = true
-	sourceFile = "source-file-path"
-	sourceImage = ""
-	family = "a-family"
-	description = "a-description"
-	network = "a-network"
-	subnet = "a-subnet"
-	region := "a-region"
-
-	got := buildDaisyVars("translate/workflow/path", imageName, sourceFile,
-		sourceImage, family, description, region, subnet, network, noGuestEnvironment)
 
-	assert.Equal(t, got["image_name"], "image-a")
-	assert.Equal(t, got["translate_workflow"], "translate/workflow/path")
-	assert.Equal(t, got["install_gce_packages"], "false")
-	assert.Equal(t, got["source_disk_file"], "source-file-path")
-	assert.Equal(t, got["family"], "a-family")
-	assert.Equal(t, got["description"], "a-description")
-	assert.Equal(t, got["import_network"], "global/networks/a-network")
-	assert.Equal(t, got["import_subnet"], "regions/a-region/subnetworks/a-subnet")
-	assert.Equal(t, len(got), 8)
-}
-
-func TestBuildDaisyVarsFromImage(t *testing.T) {
-	resetArgs()
-	imageName = "image-a"
-	noGuestEnvironment = true
-	sourceFile = ""
-	sourceImage = "source-image"
-	family = "a-family"
-	description = "a-description"
-	network = "a-network"
-	subnet = "a-subnet"
-	region := "a-region"
-
-	got := buildDaisyVars("translate/workflow/path", imageName, sourceFile,
-		sourceImage, family, description, region, subnet, network, noGuestEnvironment)
+	testCases := []struct {
```

Comment:
Split these tests back. Batching tests in one test method only makes it harder to know what actually failed without looking at unit test code. On the other hand, if we keep the "one test - one method" convention, just looking at the test method name will tell us what failed.

Reply:
Reverted